app/controllers/defendants_controller.rb

```diff
@@ -3,9 +3,9 @@
 class DefendantsController < ApplicationController
   before_action :load_and_authorize_search
 
-  add_breadcrumb 'Search filters', :new_search_filter_path
+  add_breadcrumb :search_filter_breadcrumb_name, :new_search_filter_path
   add_breadcrumb :search_breadcrumb_name, :search_breadcrumb_path
-  add_breadcrumb 'Case details',
+  add_breadcrumb (proc { |v| v.prosecution_case_name(v.controller.defendant.prosecution_case_reference) }),
```

Comment:
Not part of the actual user journey, is this  required?

Reply:
This adds the "Case ACASEREF" breadcrumb when on the defendants page so is part of the user journey. Are you referring to If the user skips and goes straight to defendant it also gets inserted - this is true but Dan liked that (and technically would require additional work to remove)